asm-multiproject-patch/Kptfile

```diff
@@ -0,0 +1,184 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: .
+openAPI:
+  definitions:
+    io.k8s.cli.setters.base-dir:
+      x-k8s-cli:
+        setter:
+          name: base-dir
+          value: base
+    io.k8s.cli.setters.gcloud.compute.network:
+      type: string
+      x-k8s-cli:
+        setter:
+          name: gcloud.compute.network
+          value: default
+    io.k8s.cli.setters.gcloud.compute.subnetwork:
+      type: string
+      x-k8s-cli:
+        setter:
+          name: gcloud.compute.subnetwork
+          value: default
+    io.k8s.cli.setters.gcloud.container.cluster.clusterSecondaryRange:
+      type: string
+      x-k8s-cli:
+        setter:
+          name: gcloud.container.cluster.clusterSecondaryRange
+          value: ""
+    io.k8s.cli.setters.gcloud.container.cluster.servicesSecondaryRange:
+      type: string
+      x-k8s-cli:
+        setter:
+          name: gcloud.container.cluster.servicesSecondaryRange
+          value: ""
+    io.k8s.cli.setters.gcloud.container.cluster.releaseChannel:
+      type: string
+      x-k8s-cli:
+        setter:
+          name: gcloud.container.cluster.releaseChannel
+          value: REGULAR
+    io.k8s.cli.setters.anthos.servicemesh.trustDomainAliases:
+      type: array
+      x-k8s-cli:
+        setter:
+          name: anthos.servicemesh.trustDomainAliases
+          listValues:
```

Comment:
can you talk with Phani to double check that array is well supported by kpt?

Reply:
Just confirmed with Phani, it is supported and we can use it.